ui/pom.xml

```diff
@@ -95,7 +95,55 @@
             <version>${gwtmockito.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>pentaho</groupId>
+            <artifactId>pentaho-platform-extensions</artifactId>
+            <version>${pentaho-scheduler-plugin.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>pentaho</groupId>
+            <artifactId>pentaho-platform-repository</artifactId>
+            <version>${pentaho-scheduler-plugin.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>pentaho</groupId>
+            <artifactId>pentaho-scheduler-core</artifactId>
+            <version>${pentaho-scheduler-plugin.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
```

Comment:
@srallapa pentaho-scheduler-plugin is a ce plugin. Is there a specific reason to have an ee dependency?